### PR TITLE
Update to 0.8.4

### PR DESCRIPTION
**Pull request by vinnyjth**
### How do I test this PR?

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-XXX](#url)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
Tested on iOS and Android. Looking good!

File: packages/newspringchurchapp/android/app/build.gradle [hunk 1]
```diff
@@ -117,12 +117,13 @@ android {
         applicationId "com.newspringchurchapp"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-versionCode 7
-        versionName "0.8.2"
+        versionCode 8
+        versionName "0.8.3"
```
Comment (richarddubay) [inline packages/newspringchurchapp/android/app/build.gradle]: This should be 7.0.0 ... according to @redreceipt

File: packages/newspringchurchapp/ios/newspringchurchapp-tvOS/Info.plist [hunk 2]
```diff
@@ -15,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.8.2</string>
+	<string>0.8.3</string>
```
Comment (richarddubay) [inline packages/newspringchurchapp/ios/newspringchurchapp-tvOS/Info.plist]: This should be `7.0.0` as well.

File: packages/newspringchurchapp/ios/newspringchurchapp-tvOSTests/Info.plist [hunk 3]
```diff
@@ -15,10 +15,10 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.8.2</string>
+	<string>0.8.3</string>
```
Comment (richarddubay) [inline packages/newspringchurchapp/ios/newspringchurchapp-tvOSTests/Info.plist]: Probably `7.0.0` again?

**Review by IsaacHardy — DISMISSED** [review]

**Review by IsaacHardy — APPROVED** [review]
